docs/jade-getting-started.md

```diff
@@ -377,24 +328,24 @@ Instead, you can select a specific test to run either in Intellij or the command
 First, make sure you have run through the following steps:
 * Start postgres
 * Ensure docker is running
-* You may need to re-auth with vault every so often. Run `vault login -method=github token=$(cat ~/.gh_token)`
-* Run `./render-configs.sh -a integration` to pull secrets from vault. For Azure Integration tests,
+* Auth as your broadinstitute.org to pull from Google Secrets Manager
```

Comment:
the old instructions say how to auth with vault maybe we should now include the auth command for google here

Reply:
Ah, sure, can do.